### ISSUE TITLE
Feature Request: GitHub Action for bitexpert/phpstan-magento

### ISSUE THREAD

**Issue by shochdoerfer** [bitExpert/phpstan-magento#49]
Build a GitHub Action to let phpstan-magento run in GitHub build pipelines automatically. Check what the extdn people have done for their actions and adapt https://github.com/extdn/github-actions-m2

**Comment by shochdoerfer** [comment]
@torhoehn did you hack on that during the Firegento Hackathon?

**Comment by torhoehn** [comment]
@shochdoerfer No, I was a little bit late, so @frostblogNet proposed this topic. As I have arrived @schmengler looked for a partner, so we worked together on something else.

**Comment by shochdoerfer** [comment]
No worries, just saw on Twitter that this topic was proposed. Wanted to avoid that we both work on this and no one is aware ;)

**Comment by torhoehn** [comment]
@shochdoerfer Is this solved, because the repo already runs GitHub Actions or do you thinking about extending this?

**Comment by shochdoerfer** [comment]
Not yet solved. The goal was to have GitHub Action with PHPStan and this extension so that others could run it easily in their GitHub CI pipeline. Similar to what the extdn are doing: https://github.com/extdn/github-actions-m2/tree/master/magento-phpstan

**Comment by torhoehn** [comment]
Ah now I remember. While writing my previous comment I wasn't in the correct context.

**Comment by shochdoerfer** [comment]
No worries, that happens ;) Still, if you feel, you wanna tackle it, let me know.

**Comment by shochdoerfer** [comment]
It could be that it is possible to use the phpstan GitHub Action provided by the extdn people: https://github.com/extdn/github-actions-m2/tree/master/magento-phpstan

Check and document!

**Comment by damienwebdev** [comment]
Solving this here 6 years late: https://github.com/graycoreio/github-actions-magento2/pull/323

**Comment by shochdoerfer** [comment]
Good to hear that there are still people using this extension :)
